spark-connect/client/src/main/scala-spark-master/io/delta/connect/tables/DeltaTable.scala

```diff
@@ -1037,6 +1037,19 @@ class DeltaTable private[tables](
  * @since 4.0.0
  */
 object DeltaTable {
+
+  /**
+   * Helper method to get the active SparkSession.
+   * 
+   * @return The active SparkSession if one exists
+   * @throws IllegalArgumentException if no active SparkSession is found
+   */
+  private def getSparkSession(): SparkSession = {
```

Comment:
```suggestion
  private def getActiveSparkSession(): SparkSession = {
```

I think `getSparkSession` can be a bit misleading